components/article/stylesheets/fullscreen_article.styl

```diff
@@ -111,36 +101,59 @@ body.body-fullscreen-article
   opacity 0.17
   min-height 500px
   z-index 1
+
 .article-fullscreen-text-overlay
   color white
+
 .article-fullscreen-video-player, .article-fullscreen-image
   object-fit cover
   z-index 0
+
 .article-container h1.article-fullscreen-title
   font-size 72px
   font-style italic
   line-height 110%
-  width 750px
   text-align center
-  @media screen and (min-width 1100px)
-    width 900px
+
 .article-fullscreen.article-sa-fullscreen
   h1
     margin-bottom 60px
     text-align left
   .main-layout-container
     margin-top 0px
+
 .article-fullscreen-intro.article-lead-paragraph
   font-size 22px
   line-height 120%
-  width 750px
   margin-bottom 0px
-  @media screen and (min-width 1100px)
-    width 900px
-  @media screen and (min-width 1500px)
-    font-size 25px
 
-.article-sa-fullscreen-logos
-  width 750px
+.article-fullscreen
```

Comment:
Just trying to clean up this file by moving all thing media queries to the bottom.